.github/workflows/verify_license.yml

```diff
@@ -30,8 +30,8 @@ jobs:
           go-version-file: './go.mod'
           check-latest: true
       - name: Install addlicense
-        run: go install github.com/google/addlicense@v1.0.0
+        run: go install github.com/google/addlicense@latest
```

Comment:
Any specific reason for binding this to `latest` and not a git OID?

Reply:
There's rarely updates, and dependabot also won't pick up updates for this since it's part of the command. I pinned it to v1.1.1